README.md

```diff
@@ -203,13 +203,19 @@ Add a function named `highestScoreFrom` in the `Adagrams` object in `src/adagram
 
 #### Process
 
-This Optional Wave has 3 parts:
+This Wave has 3 parts:
 
 1. Adjust the syntax for the tests for `highestScoreFrom` to run, instead of skip. To do this, find the `describe` block for the tests of `highestScoreFrom`, and change the syntax from `describe.skip` to `describe`
 2. Write the pseudocode for this function, using whatever resources and references
 3. Translate the pseudocode into JavaScript, using whatever resources and references
 
-You are encouraged the leave the pseudocode as comments for this wave if you begin this wave and don't finish it. The goal for this optional wave is to practice translating logic to new JavaScript syntax.
+#### What if I Might Not Finish This Wave?
+
+You are encouraged the leave the pseudocode as comments for this wave if you begin this wave and don't finish it. The goal for this wave is to practice translating logic to new JavaScript syntax.
+
+If reviewers see your practice and thought process with pseudocode, they will review accordingly with respect to the practice and work put into the project overall, even if this wave isn't finished.
```

Comment:
Maybe add, "If you aren't able to complete this part, please reflect on you process and where you got stuck in the reflection (comprehension?) questions when you submit to project"